_posts/2021-03-24-sncom-design-sprint.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "6 ways to unlock true collaboration with a headless cms"
+author: [molnarm84, huszerlbence]
+image: "../img/posts/miro_full.png"
+tags: [productivity, teamwork, collaboration, saas, headless cms, sensenet]
+redirect_to: https://www.sensenet.com/blog/sncom-design_sprint
+---
+
+6 people, 4 days, ∞ coffees and the result is 1 prototype and lot of feedbacks and new ideas by the end of the week. This was our design sprint in a nutshell.
+
+---
+
+## Roots
+
+We launched our new website in October 2020 because we came out with our SaaS model (called sensenet as a service) currently in BETA. This new site with fresh content for the new era allowed us to lead visitors to register to the service. Back then we must to act fast since we wanted to get the whole thing ready for Hactoberfest, for this reason some processes was not detailed enough. Since last year we gathered some stats about the usage and interaction with our site and we thought that some thing can and should be improved.
+
+## Goal
+
+We have some early birds registered to the BETA but we still need more feedback in this phase so our goal is to convince more people to give SNaaS a try. Since this is a much bigger task to fit into a simple 1,5 hours meeting, a design sprint seemed as a perfect solution to accomplish this challenge. Our objective was to start from square one and go through the whole process and fine-tune the journey of a customer.
```

Comment:
```suggestion
We have some early birds registered to the BETA but we still need more feedback in this phase so our goal is to convince more people to give SNaaS a try. Since this is a much bigger task to fit into a simple 1,5 hours meeting, a [design sprint](https://www.thesprintbook.com/how) seemed as a perfect solution to accomplish this challenge. Our objective was to start from square one and go through the whole process and fine-tune the journey of a customer.
```